Packs/AbuseDB/pack_meta.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AbuseIPDB",
+    "description": "Central repository to report and identify IP addresses that have been associated with malicious activity online. Check the Detailed Information section for more information on how to configure the integration.",
+    "support": "xsoar",
```

Comment:
```suggestion
    "support": "Cortex XSOAR",
```

Reply:
should be `xsoar`, it is different from `author` field

Reply:
We have a few which are using this for support. We should change those then.

Reply:
Yes, and also change it in demisto-sdk if it was not changed yet

Reply:
I'll include it in my PR

Reply:
o.k I think it is defined here: https://github.com/demisto/etc/issues/23216#issuecomment-608375523

Reply:
sdk is quite out of date:

```
        metadata = {
            'name': '## FILL OUT MANUALLY ##',
            'description': '## FILL OUT MANUALLY ##',
            'support': 'demisto',
            'serverMinVersion': '## FILL OUT MANUALLY #',
            'currentVersion': PACK_INITIAL_VERSION,
            'author': 'demisto',
            'url': 'https://www.demisto.com',
            'email': '',
            'categories': [],
            'tags': [],
            'created': datetime.utcnow().strftime(Initiator.DATE_FORMAT),
            'updated': datetime.utcnow().strftime(Initiator.DATE_FORMAT),
            'beta': False,
            'deprecated': False,
            'certification': 'certified',
            'useCases': [],
            'keywords': [],
            'price': '0',
            'dependencies': {},
        }
```

I'll update the other fields while I'm at it.